Doc/reference/lexical_analysis.rst

```diff
@@ -254,20 +256,34 @@ Whitespace between tokens
 Except at the beginning of a logical line or in string literals, the whitespace
 characters space, tab and formfeed can be used interchangeably to separate
 tokens.  Whitespace is needed between two tokens only if their concatenation
-could otherwise be interpreted as a different token (e.g., ab is one token, but
-a b is two tokens).
+could otherwise be interpreted as a different token. For example, ``ab`` is one
+token, but ``a b`` is two tokens. However, ``+a`` and ``+ a`` both produce
+two tokens, ``a`` and ``b``, as ``+a`` is not a valid token.
```

Comment:
Thanks for the suggested new wording, but I think there's an error:
```suggestion
token, but ``a b`` is two tokens. However, ``+a`` and ``+ a`` both produce
two tokens, ``+`` and ``a``, as ``+a`` is not a valid token.
```

Reply:
:man_facepalming:

Reply:
Thanks for the catch!